fellowship/index.html

```diff
@@ -452,7 +452,12 @@ <h4>What are the guest speakers like?</h4>
 						<br/>
 						<h4>What’s the application process like?</h4>
 						<p>
+<<<<<<< HEAD
+								Applications for the 2020 Impact Fellowship are currently <a target="_blank" href=https://airtable.com/shr8e22oD86uvFQjF>live!</a>
+								The application should take no more than 20 minutes to complete, and fellows are admitted on a rolling basis. The deadline to apply is December 1st.								
+=======
 							Applications for the 2020 Impact Fellowship are now closed. <a target="_blank" href="https://impactlabs.us19.list-manage.com/subscribe/post?u=e271c51cea784ba1c111d990e&amp;id=90db606254">Subscribe to our newsletter</a> to be notified when applications for the next year's fellowship open.
+>>>>>>> 5c85f181b9fef5bdcdc09fec819f09d8d19a1e3d
```

Comment:
These are from a merge conflict, please resolve first and run the site locally to check.

Reply:
@thejameswang